src/background.js

```diff
@@ -104,14 +104,20 @@ function processPRs(prList, useCreated) {
  */
 function getPRs(force) {
   let currentPrs, useCreatedFlag;
-  return Promise.all([
-    getGH(),
-    getChromeStorage(config.githubUserName),
```

Comment:
this was assuming that the github username was always set in options...which its not